tls/s2n_security_policies.h

```diff
@@ -191,3 +192,10 @@ int s2n_find_security_policy_from_version(const char *version, const struct s2n_
 int s2n_validate_kem_preferences(const struct s2n_kem_preferences *kem_preferences, bool pq_kem_extension_required);
 S2N_RESULT s2n_validate_certificate_signature_preferences(const struct s2n_signature_preferences *s2n_certificate_signature_preferences);
 S2N_RESULT s2n_security_policy_get_version(const struct s2n_security_policy *security_policy, const char **version);
+S2N_RESULT s2n_security_policy_validate_certificate(const struct s2n_cert_info *info,
+        const struct s2n_security_policy *security_policy);
+/* Checks to see if a certificate has a signature algorithm that's in our 
+ * certificate_signature_preferences list 
+ */
+S2N_RESULT s2n_security_policy_validate_sig_scheme_supported(const struct s2n_cert_info *info,
```

Comment:
nit: Since this is a s2n_security_policy function, I feel like the security policy argument should come first. Could just be me though.